### PR TITLE
Versioning bundle

### PR DESCRIPTION
**Pull request by MatiasOS**
- update webpack prod config to add hash to bundle name
- add a new file index.templete.html 
- update npm start script similar to prod script.

@skkr please check if is all ok with the new template and if all css is still ok.

### CODE REVIEW

**Review by skkr — APPROVED** [review]
Looks good. Just to keep in mind...

We should add the meta tags for social networks in the new template, and after merging the styleguide branch we'll need to remove the link tags from it, since the CSS files are imported from settings.css.

**Review by AugustoL — APPROVED** [review]

**Comment by kvbik** [comment]
so far perfect! let's use the same cloudflare purge script as in https://github.com/windingtree/wt-demo-app/pull/100

File: package.json [hunk 1]
```diff
@@ -5,13 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --config ./webpack/dev/webpack.dev.config.js --content-base --public --hot --host 0.0.0.0 --disable-host-check",
-    "dev": "webpack-dev-server --config ./webpack/dev/webpack.dev.config.js --content-base --public --host 0.0.0.0 --disable-host-check",
-    "build": "npm run clean && webpack --config ./webpack/prod/webpack.prod.config.js",
+    "start": "npm run build && serve --single ./public",
```
Comment (JirkaChadima) [inline package.json]: This is actually not a good idea. Now.sh runs `npm run build` automatically before running the start command. With this configuration, the deployment will take twice as long.

File: src/index.template.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <head>
+    <title> Lif Market Validation Mechanism - WindingTree </title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <link  rel="stylesheet" href="https://fonts.googleapis.com/css?family=Catamaran:400,500,600,700,800">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/ShakrMedia/tuesday/master/build/tuesday.min.css" />
+
+    <link rel="stylesheet" href="app.css">
```
Comment (JirkaChadima) [inline src/index.template.html]: The `app.css` shouldn't be necessary. The `HtmlWebpackPlugin` should inject CSS automatically, if there's at least one place where the main css file is required.

File: package.json (hunk 1 above)
Comment (MatiasOS) [reply package.json]: Done! Add `now-start` to serve only.

File: src/index.template.html (hunk 2 above)
Comment (MatiasOS) [reply src/index.template.html]: Yes, removed!

**Comment by MatiasOS** [comment]
@kvbik Added script to clean CloudFlare cache
@skkr Please, check if is all ok in `index.template.js`.from now on we will only use  `index.template.html` to avoid duplicate code

File: webpack/prod/postcss.config.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ module.exports = ({
         }
       }),
       destination: 'public/styleguide/',
-      index: 'index.html',
+      index: 'index.template.html',
```
Comment (skkr) [inline webpack/prod/postcss.config.js]: This index file is not the same as the app index, but the index in /styleguide , we should keep the original name.
The changes on the template look good.

File: .travis.yml [hunk 4]
```diff
@@ -14,4 +14,12 @@ jobs:
     - npm run alias
 env:
   global:
-    secure: T0y2j5vGQIGoLWa/6uLngEsFGff14fe4xQezkltLp5KG1oJCgcQ8klpDvGGR+MphtDgH0NvC6ALHLqEwhnxfuhII+9AFhJic8B65Ww19Z7Hl4ZCggDG+CchwApJmI2SyJXgOXIOVz0kj1mR3wNkgu+bmPxQmGOIP3QlB+IuiZqxrWwNUjjMZzg5lFSRQxlR2JTxANDRMi25PgXfwNHgmzghGhHGKrKMRtp6+A6K+gUmmZvQt5I5xOnmQfpbwEIj805nsJHit3iTZ9yRJ3lzg1b/CkhfvFkS3JS7rDpuFSl6f5r8lI4zpQmTA2V9NyqREgZ+Qu1y5yA11NB3/mPsh4M2FB5+lH64jrYKHBYEaQH59lmfyAxNNXAR7QJldXNzBu/ljBXy/kb+W8wPBCTynXV8kGmMPykLNs+4+cim6avg1Fjb6E9KPAHo6nCFi4OvzrOHjs9UsoMVsH/0dXDi8QCSsPjGwuvgRPP3Oz5QA+r4H1Xm7FqUeAR6bH8dVzeZFT5UbiMBwkjM6fgN+DAVqPzPv8au4qkWrzs7oWLH/uSGwqsXOy6nRvT/CqkzdXdtz2qzEQ2cG1lola7eBMMBjmdKjdtBK295tIgWovFqUF6LtOo7/Ytq0jthTU1sunuIJoBuWcSWBM0ayIL6oIxzCb0APcsw+Ad/+TCzmjUymEJU=
+  - CF_ROOT=https://demo.windingtree.com
```
Comment (JirkaChadima) [inline .travis.yml]: @MatiasOS This has to say `https://mvm.windingtree.com`

Comment (MatiasOS) [reply .travis.yml]: fixed now

**Pull request merged**